Library/Homebrew/env_config.rb

```diff
@@ -397,6 +397,10 @@ module EnvConfig
                      "from homebrew-core.",
         boolean:     true,
       },
+      HOMEBREW_NO_WARN_OUTDATED_XCODE:           {
+        description: "If set, Homebrew will not warn about using outdated (but still compatible) Xcode versions.",
+        boolean:     true,
+      },
```

Comment:
I'm inclined not to document this at all:
```suggestion
```
in which case you should ignore my other suggestion about `Homebrew::EnvConfig` (since that relies on the env variable being in `env_config.rb`).